x/stakeibc/module_ibc.go

```diff
@@ -164,6 +168,10 @@ func (im IBCModule) OnTimeoutPacket(
 	modulePacket channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) error {
+	err := im.keeper.ICACallbacksKeeper.CallRegisteredICACallback(ctx, modulePacket, []byte{})
```

Comment:
It's the latter - there might be some callback logic that we want to execute in case of a timeout. Agreed, this should be better documented. Added to the PR description for now, it might be worth adding a README to `icacallbacks` as part of this PR - wdyt?